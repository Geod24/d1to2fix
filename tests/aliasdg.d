alias void delegate ( int ) SomeDg;

void foo ( SomeDg dg1, SomeDg d2 );
