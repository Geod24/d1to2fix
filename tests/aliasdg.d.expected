alias void delegate ( int ) SomeDg;

void foo ( scope SomeDg dg1, scope SomeDg d2 );
